config/browse_everything_providers.yml

```diff
@@ -18,3 +18,9 @@ file_system:
 # sky_drive:
 #   :client_id: YOUR_MS_LIVE_CONNECT_CLIENT_ID
 #   :client_secret: YOUR_MS_LIVE_CONNECT_CLIENT_SECRET
+# sharepoint:
+#   client_id: YOUR_MICROSOFT_GRAPH_CLIENT_ID
+#   client_secret: YOUR_MICROSOFT_GRAPH_CLIENT_SECRET
+#   tenant_id: YOUR_MICROSOFT_GRAPH_TENANT_ID
+#   scope: offline_access https://graph.microsoft.com/.default
+#   redirect_uri: https://example.com/browse/connect
```

Comment:
The `redirect_uri` param should be the application's base url appended with `/browse/connect`. So Avalon-dev should be `https://avalon-dev.dlib.indiana.edu/browse/connect`. This same value should be what is entered in the `redirect_uri` setting for the application in the Entra admin center.